internal/svcmanager/svcmanager.go

```diff
@@ -20,20 +21,27 @@ import (
 // MinimalTimeout is the minimum timeout required for requests that are meant to be near-instant
 const MinimalTimeout = 500 * time.Millisecond
 
+var errVersionMismatch = locale.NewError("err_ping_version_mismatch")
+
 type Manager struct {
-	ready bool
-	cfg   configurable
+	ready        bool
+	checkVersion bool
+	cfg          configurable
 }
 
 type configurable interface {
 	GetInt(string) int
 }
 
 func New(cfg configurable) *Manager {
-	mgr := &Manager{false, cfg}
+	mgr := &Manager{false, true, cfg}
 	return mgr
 }
 
+func (m *Manager) SetCheckVersion(check bool) {
+	m.checkVersion = check
+}
```

Comment:
This isn't used (nor should it be).

Reply:
Whoops, meant to remove this and the `bool`